.github/workflows/ci-test.yaml

```diff
@@ -114,8 +114,6 @@ jobs:
             clients/js/lib
 
       - name: Test JS client
-        env:
-          NODE_OPTIONS: "--no-experimental-fetch" # https://github.com/nock/nock/issues/2397
```

Comment:
Was `nock` and `node-fetch` removed from dependencies, as this was clearly there, to bypass node fetch, in order for `nock` to work?

Reply:
After removal of the code which conditionally imports `http` or `https` modules Nock seems to be working - with node v18 which has built-in `fetch` support.

We only support node v18+

Reply:
I am referring to why are `nock` and `nock-fetch` still listed in dependencies, if we are now using only node:

https://github.com/oasisprotocol/sapphire-paratime/blob/2155379b2207aefb9bbf9d2f6850be4601e7bb64/clients/js/package.json#L62-L63